microsoft/testsuites/vm_extensions/run_command.py

```diff
@@ -0,0 +1,295 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import uuid
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+
+from lisa.sut_orchestrator.azure.common import (
+    AZURE_SHARED_RG_NAME,
+    get_storage_account_name,
+    get_or_create_storage_container,
+    generate_blob_sas_token,
+)
+
+from assertpy import assert_that
+
+from lisa.sut_orchestrator.azure.features import AzureExtension
+from lisa.sut_orchestrator.azure.platform_ import AzurePlatform
+from lisa.environment import Environment
+
+
+@TestSuiteMetadata(
```

Comment:
Please leave only one test case on P1, others should be P2 or P3.

Reply:
Can you leave one most important and fast test case to P1? So it will be included to wider test matrix?